### PR TITLE
chore: auto-closed status setting modal

### PR DESCRIPTION
**Pull request by jhd2best**
## What does this PR do?
as https://github.com/lensterxyz/lenster/issues/1844 request. we add auto-closed status setting bar feature in this PR

image

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking small changes to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?
- [ ] Login to Lenster
- [ ] Navigate to the top right corner, click on the display picture
- [ ] Select the option "Set status" from the drop down (will open a dialog box)
- [ ] Update the required status in the box and click on "Save"
- [ ] Check whether status setting modal auto-closed

## Checklist
- I have read the [contributing guide](https://github.com/lensterxyz/lenster/blob/main/CONTRIBUTING.md)
- My code follow the style guidelines of this project
- I have performed a self-review of my own code and corrected any misspellings
- I have commented my code, particularly in hard-to-understand areas
- I have checked if my PR needs changes to the documentation
- I have checked if my changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- I have checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/src/components/Shared/Status.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ import { v4 as uuid } from 'uuid';
 import { useContractWrite, useSignTypedData } from 'wagmi';
 import { object, string } from 'zod';
 
+import { useGlobalModalStateStore } from '../../store/modals';
```
Comment (bigint) [inline apps/web/src/components/Shared/Status.tsx:32]: better use `import { useGlobalModalStateStore } from 'src/store/modals';`

**Review by bigint — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -45,6 +45,9 @@ yarn-error.log*
 # playwright
 .gitsigners
 
+# jetbrains IDE
```
Comment (bigint) [inline .gitignore:48]:
```suggestion
# JetBrains IDE
```

**Comment by bigint** [comment]
Closes T-6926

File: apps/web/src/components/Shared/Status.tsx (hunk 1 above)
Comment (jhd2best) [reply apps/web/src/components/Shared/Status.tsx]: sure. updated

**Review by bigint — APPROVED** [review]
**LGTM** :shipit: 

Rad LGTM

**Pull request merged**